### ISSUE TITLE
Bug: Horizontal divider in quick-switcher is a selectable item

### ISSUE THREAD

**Issue by redactedscribe** [Lisandra-dev/obsidian-create-note-in-folder#12]
A very minor bug, but the horizontal dividing line in the quick-switcher is an item: Pressing down arrow past the user-defined folders first lands on the line, and then another press moves on to the first non-user-defined folder. The good news is nothing bad happens if you press Return on the line (the selected line just jumps back to the top). Something you may want to one day fix.

Thanks.

**Comment by Mara-Li** [comment]
Yeah, I know. I was verry bored by this line, so I made some a hack because I don't know how to fix this. That's why it returns to the first line.
